src/sweeps/config/schema.json

```diff
@@ -808,6 +808,11 @@
           "description": "Metric value to use in bayes search for runs that fail, crash, or are killed",
           "enum": ["latest", "best", "worst"],
           "default": "worst"
+        },
+        "impute_while_running": {
```

Comment:
does this depend on the impute value being set? What happens when impute isn't set but this is?

Reply:
see above comment. we might want to let this be `"enum": ["latest", "best", "worst", false]` with `default: false`

Reply:
use an actual `false` here instead of `"false"`. enums let you use multiple types in an enum:

https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values

Reply:
can you also please add an exception that hard fails `/search` if we get a value of `impute_while_running` that is not one of the allowed four? we should add to this function here: 

https://github.com/wandb/sweeps/blob/master/src/sweeps/bayes_search.py#L25